docs/root/modules/index.rst

```diff
@@ -1,5 +1,4 @@
 .. toctree::
-    :hidden:
```

Comment:
Removing this so that the table of contents shows up in the rendered modules github page